uvm/servlets/setup/app/view/Util.js

```diff
@@ -31,8 +31,14 @@ Ext.define('Ung.Setup.Util', {
             success: function (response) {
                 // Ung.app.loading(false);
                 if (response.responseText && response.responseText.indexOf('loginPage') != -1) {
-                    Ext.MessageBox.alert('Authentication failed'.t(), 'Invalid password.'.t());
-                    return;
+                    //Check default password, if true callback otherwise existing flow
+                    if (password == "passwd") {
+                        cb(true);
```

Comment:
But are we really reaching this (under success) if the authentication fails?  

I think it hits the failure section which is where you would want to do the prompt for password.

Reply:
@cblaise-untangle alert is raised under the success, i have tested this and Invalid Password Prompt is raised, as mentioned in ticket
`Ext.MessageBox.alert('Authentication failed'.t(), 'Invalid password.'.t());`